lib/user.js

```diff
@@ -108,7 +111,8 @@ function access(permission, account, callback) {
 
   // If permissions is already loaded, use them.
   if (account.permissions) {
-    callback(null, account.permissions.indexOf(permission) > -1);
+    findPermission(account.permissions);
```

Comment:
This could return a boolean captured in a local ´permissionFound´ variable and passed to the ´callback()´